### DISCUSSION TITLE
Paramètre de sécurité switch

### DISCUSSION THREAD

**Discussion by bigou56** [jmcollin78/versatile_thermostat#704] [General]
Bonjour à tous,
J'ai installé Versatile pour gérer mes radiateurs.
Cela marche très bien sauf pour mon sèche serviette qui est commandé par un switch.
J’arrive bien à faire le thermostat mais au bout de quelques minutes voir secondes tout se coupe et cela indique qu’il n’est pas en sécurité… et le radiateur passe son temps a s’éteindre puis repartir…
En cherchant dans les paramètres (voir photo) je trouve bien 2 paramètres sur la sécurité mais j’avoue ne pas bien comprendre comment les régler.
![IMG_0883](https://github.com/user-attachments/assets/253070f0-21c6-4fae-a438-fbec8486c8e4)
![IMG_0882](https://github.com/user-attachments/assets/7e1e5722-7c6a-45ef-845d-5621e6d3e6d3)
![IMG_0881](https://github.com/user-attachments/assets/892a0b55-f358-4b29-bcb7-dda4f1d673f4)
Merci de votre aide,

**Comment by jmcollin78** [comment]
Hello, regarde ici : https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/fr/troubleshooting.md#pourquoi-mon-versatile-thermostat-se-met-en-securite-
